shared/actions/chat/thread-content.js

```diff
@@ -176,7 +195,7 @@ function* _loadMoreMessages(action: ChatGen.LoadMoreMessagesPayload): Saga.SagaG
         yield Saga.put(ChatGen.createThreadLoadedOffline({conversationIDKey}))
       }
 
-      yield Saga.put(ChatGen.createSetLoaded({conversationIDKey, isLoaded: !error})) // reset isLoaded on error
+      yield Saga.put(ChatGen.createSetLoaded({conversationIDKey, isLoaded: true}))
```

Comment:
This fixes the problem where we mark as not loaded things that errored out which could cause an infinite loop of actions loading over and over. The flag does get reset if the error is actually fixed by the core side telling us we should clear the messages